tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -258,14 +264,14 @@ public List<VectorTile.Feature> postProcess(int zoom, List<VectorTile.Feature> i
 
     List<VectorTile.Feature> top64 = cities.subList(startIndex, endIndex);
 
-    // This should always be less than 64 now that a label grid is being used
+    // This should always be less than 32 now that a label grid is being used
```

Comment:
Something is still off in this section... if I take it down to 32 from 64 some big cities still drop out in error... which suggests an error in the sort algo?